src/SDKs/ApplicationInsights/Data.ApplicationInsights/Microsoft.Azure.ApplicationInsights.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
```

Comment:
Please add 
```
<!-- Please do not move/edit code below this line -->
	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
<!-- Please do not move/edit code below this line -->
``` 
at the top and 
```
<!-- Please do not move/edit code below this line -->
	<Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
<!-- Please do not move/edit code above this line -->
```

at the bottom of this file, as [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/ApplicationInsights/Management.ApplicationInsights/Microsoft.Azure.Management.ApplicationInsights.csproj#L2)